script.rb

```diff
@@ -41,7 +41,7 @@
 		iconType = ""
 	end
 	# Assemble this item's XML string for Alfred. See http://www.alfredforum.com/topic/5-generating-feedback-in-workflows/
-	thisXmlString = "\t<item uid=\"#{processName}\" arg=\"#{processId}\">
+	thisXmlString = "\t<item arg=\"#{processId}\">
```

Comment:
Make this configurable.

Also don't remove the uid but instead conditionally use [`skipknowledge`](https://www.alfredapp.com/alfred-5-whats-new/#:~:text=The%20Script%20Filter%20object%20has%20a%20new%20%60skipknowledge%60%20boolean%20field%20in%20the%20JSON%2C%20preventing%20Alfred%20from%20sorting%20your%20outputted%20results%20when%20using%20UIDs%20for%20reselection) to show the most killed processes before that the user starts typing a process name.